packages/gdrcopy.spec

```diff
@@ -150,9 +150,19 @@ cp -a $RPM_BUILD_DIR/%{name}-%{version}/src/gdrdrv/Makefile $RPM_BUILD_ROOT%{usr
 cp -a $RPM_BUILD_DIR/%{name}-%{version}/src/gdrdrv/nv-p2p-dummy.c $RPM_BUILD_ROOT%{usr_src_dir}/gdrdrv-%{version}/
 cp -a $RPM_BUILD_DIR/%{name}-%{version}/dkms.conf $RPM_BUILD_ROOT%{usr_src_dir}/gdrdrv-%{version}
 
+%if 0%{!?suse_version:1}
 # Install gdrdrv service script
 install -d $RPM_BUILD_ROOT/etc/init.d
 install -m 0755 $RPM_BUILD_DIR/%{name}-%{version}/init.d/gdrcopy $RPM_BUILD_ROOT/etc/init.d
+%else
+mkdir -p $RPM_BUILD_ROOT/etc/modprobe.d
+cat <<"EOF" > $RPM_BUILD_ROOT/etc/modprobe.d/50-gdrdrv.conf
+#options gdrdrv dbg_enabled=1
+#options info_enabled=1
+install gdrdrv PATH=$PATH:/bin:/usr/bin; /sbin/modprobe -i gdrdrv && rm -f /dev/gdrdrv && mknod -m 666 /dev/gdrdrv c $(gawk '/gdrdrv/{printf"%%s",$1}' /proc/devices) 0 && chgrp video /dev/gdrdrv
```

Comment:
just wondering if you might need `--ignore-install` here.

Reply:
Shouldn't  `-i` and `--ignore-install` should essentially be the same in this context? Since the man page explicitly uses it in its example I will replace it. $CMDLINE_OPTS should definitely be added.

Reply:
you are right, `-i` and `--ignore-install` are synonymous.